### PR TITLE
golangci-lint: update to 1.55.2

### PR DESCRIPTION
**Pull request by steenzout**
#### Description

* update [golangci-lint](https://github.com/golangci/golangci-lint) to [v1.55.2](https://github.com/golangci/golangci-lint/releases/tag/v1.55.2)

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 14.2.1
Xcode 15.2 / Command Line Tools x.y.z

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?

```shell
port -vst install golangci-lint
--->  Cleaning golangci-lint
--->  Removing work directory for golangci-lint
--->  Scanning binaries for linking errors
--->  No broken files found.
--->  No broken ports found.
```

- [x] tested basic functionality of all binary files?

```shell
$ command -v golangci-lint
/opt/local/bin/golangci-lint
```

```shell
$ golangci-lint --help   
Smart, fast linters runner.

Usage:
  golangci-lint [flags]
  golangci-lint [command]

Available Commands:
  cache       Cache control and information
  completion  Generate the autocompletion script for the specified shell
  config      Config
  help        Help
  linters     List current linters configuration
  run         Run the linters
  version     Version

Flags:
      --color string              Use color when printing; can be 'always', 'auto', or 'never' (default "auto")
  -j, --concurrency int           Concurrency (default NumCPU) (default 10)
      --cpu-profile-path string   Path to CPU profile output file
  -h, --help                      help for golangci-lint
      --mem-profile-path string   Path to memory profile output file
      --trace-path string         Path to trace output file
  -v, --verbose                   Verbose output
      --version                   Print version

Use "golangci-lint [command] --help" for more information about a command.
```

- [x] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: devel/golangci-lint/Portfile [hunk 1]
```diff
@@ -19,12 +19,12 @@ long_description    \
 categories          devel
 installs_libs       no
 license             GPL-3
-maintainers         {gmail.com:herby.gillot @herbygillot} \
+maintainers         {gmail.com:herby.gillot @herbygillot @steenzout} \
```
Comment (herbygillot) [inline devel/golangci-lint/Portfile:22]: If you'd like to add yourself as co-maintainer, you need to add your Github handle on its own line:

```suggestion
maintainers         {gmail.com:herby.gillot @herbygillot} \
                    {@steenzout} \
```

Ideally you could also add your obfuscated email address as well.

Comment (steenzout) [reply devel/golangci-lint/Portfile]: @herbygillot committed suggestion. I left my email... it wouldn't be obfuscated enough. 😄

Comment (steenzout) [reply devel/golangci-lint/Portfile]: thank you for your review! 🙏🏻

**Comment by herbygillot** [comment]
Thanks for your submission @steenzout. In the future, squash additional commits into the one original PR commit.  Thank you.

**Pull request merged**